lib/templates/immutable.yaml.mustache

```diff
@@ -19,6 +19,10 @@
       region: '{{=<% %>=}}{{ region }}<%={{ }}=%>'
       aws_zone: '{{=<% %>=}}{{ region }}<%={{ }}=%>{{=<% %>=}}{{ aws_zone }}<%={{ }}=%>'
       group: '{{{ security_group }}}'
+      {{#using_vpc}}
+      vpc_subnet_id: '{{{ vpc_subnet_id }}}'
+      assign_public_ip: yes
```

Comment:
When using classic, you don't assign a vpc, and `assign_public_ip` defaults to yes. Should `assign_public_ip: yes` go in anyway?